ui/src/app/modules/projects/modules/logs/components/log/filter/filter.component.ts

```diff
@@ -30,11 +32,11 @@ export class FilterComponent implements OnInit {
         });
     }
 
-    public getStatusLog() {
-        this.logService.statusLog().subscribe((status: any) => {
-            this.statusLog = status;
-        });
-    }
+    // public getStatusLog() {
```

Comment:
@pabloferrari1789 Maybe we can remove commented code if we are no using any more.

Reply:
Done